File: src/app/models/enums/secciones.ts
export enum Secciones {
    peru         = 1,
    guatemala    = 2,
    otherstories = 3,
    portraits    = 4,
    palimpsestos = 5,
    videos       = 6,
}

export const SeccionesLabel = new Map<number, string>([
    [Secciones.peru, 'Peru'],
    [Secciones.guatemala, 'Guatemala'],
    [Secciones.otherstories, 'Other Stories'],
    [Secciones.portraits, 'Portraits'],
    [Secciones.palimpsestos, 'Palimpsestos'],
    [Secciones.videos, 'Videos'],
]);

File: GalleryProject.Common/Enums/Secciones.cs
using System.ComponentModel;

namespace GalleryProject.Common.Enums
{
    public enum Secciones : byte
    {
        [Description("Peru")]
        Peru = 1,
        [Description("Guatemala")]
        Guatemala = 2,
        [Description("Other Stories")]
        Otherstories = 3,
        [Description("Portraits")]
        Portraits = 4,
        [Description("Palimpsestos")]
        Palimpsestos = 5,
        [Description("Videos")]
        Videos = 6,
    }
}
